web_app/webutils/config.py

```diff
@@ -0,0 +1,106 @@
+
+"""
+Configuration settings for AutoG-S Web App
+Based on the actual AutoG-S backend pipeline
+"""
+
+# LLM Model Configurations (matching main.autog_s exactly)
+LLM_MODELS = {
```

Comment:
Let's only use Sonnet4 and Sonnet4.5, because the rest have been warned to be out of support.